Code-4-Community/lucys-love-bus-frontend
# EventsListing responsiveness

## Issue

Closes #162 

## Changes

Adds responsiveness to the event listings.

## Verification Steps

Go to the upcoming events (and my events) pages and ensure that an event listing is cleanly responsive. Add a few events with various pictures and titles to make sure.
